src/general.scss

```diff
@@ -0,0 +1,9 @@
+:root,
+.light-app-theme,
+.default-app-theme {
+  --theme-box-shadow: 50, 51, 56;
```

Comment:
why not in monday-ui-style?

Reply:
no need, no one is using it, it is here just for history